front/src/applications/operationalStudies/components/Project/addOrEditProjectModal.scss

```diff
@@ -0,0 +1,4 @@
+.project-edition-modal {
```

Comment:
No need to create a new file I think, you can use directly _AddAndEditProjectModal.scss file :)

Reply:
obviously.. done!